File: app/src/main/java/com/apanda/base/Utils/GpsUtils.java
package com.apanda.base.Utils;

import android.content.Context;
import android.location.LocationManager;
import android.provider.Settings;

/**
 * Created by Android on 2016/9/29.
 */

public class GpsUtils {

    public static boolean GPSOpened(Context instance) {
        return Settings.Secure.isLocationProviderEnabled(instance.getContentResolver(),
                LocationManager.GPS_PROVIDER);
    }

}

File: app/src/main/java/com/apanda/base/Utils/GpsUtils.kt
package com.apanda.base.Utils

import android.content.Context
import android.location.LocationManager
import android.provider.Settings

/**
 * Created by Android on 2016/9/29.
 */

object GpsUtils {

    fun GPSOpened(instance: Context): Boolean {
        return Settings.Secure.isLocationProviderEnabled(instance.contentResolver,
                LocationManager.GPS_PROVIDER)
    }

}